### PR TITLE
Update ChangeLog to include v0.2.0

### PR DESCRIPTION
**Pull request by tedsuo**
I believe I captured all of the major changes. Did I miss anything?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,10 +7,29 @@ the release.
...
+- [Metrics Observers API](https://github.com/open-telemetry/oteps/blob/master/text/0008-metric-observer.md)
+- [Metric Handle API](https://github.com/open-telemetry/oteps/blob/master/text/0009-metric-handles.md)
+- [Rename "Cumulative" to "Counter"](https://github.com/open-telemetry/oteps/blob/master/text/0010-cumulative-to-counter.md)
+- [Named Tracers and Meters](https://github.com/open-telemetry/oteps/blob/master/text/0016-named-tracers.md)
 
-### Added
+## v0.1.0 (06-21-2019)
 
-- API proposal for the converged OpenTracing/OpenCensus project is complete.
+### Specification Changes
```
Comment (iredelmeier) [inline CHANGELOG.md]: Any particular reason for moving from `Added` (and presumably eventually `Removed`, `Changed`, etc.) in place of `Specification Changes`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,10 +7,29 @@ the release.
...
+- Removed SpanData interface in favor of Span Start and End options.
+- Added a TracerFactory interface for labeling spans library 
+- Added design goals and requirements for a telemetry data exchange protocol.
+- Added a Span Processor interface for intercepting span start and end invocations.
+- Added a Span Exporter interface for processing batches of spans.
+- Replace DistributedContext.GetIterator with GetEntries.
+- Clarifications and adjustments to improve cross-language applicability.
 
-## 0.1.0 (API proposal complete) - 06-21-2019
+### Related OTEPS
```
Comment (iredelmeier) [inline CHANGELOG.md]: Thoughts on linking these inline to the related changes?

**Comment by SergeyKanzhelev** [comment]
Also fix a build please

**Comment by SergeyKanzhelev** [comment]
@tedsuo ping

**Comment by tedsuo** [comment]
@iredelmeier @SergeyKanzhelev 
Here you go! I consolidated the OTEPs into the list, as requested. I agree, it looks better this way. 

I did not feel like most changes would easily land in only one of add/removed/changed, so I just kept it as a single list. Open to better formatting though.

**Comment by SergeyKanzhelev** [comment]
Active for more than 2 days. Have enough reviews. And not a semantics change. Merging

**Pull request merged**